tests/CMakeLists.txt

```diff
@@ -55,6 +55,14 @@ blt_add_executable(NAME blt_gtest_smoke
 blt_add_test( NAME blt_gtest_smoke
               COMMAND blt_gtest_smoke)
 
+blt_add_executable(NAME blt_gtest_death_test_smoke
+                   SOURCES blt_gtest_death_test_smoke.cpp
+                   OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                   DEPENDS_ON gtest)
+
+blt_add_test( NAME blt_gtest_death_test_smoke
+              COMMAND blt_gtest_death_test_smoke)
+
 ################
```

Comment:
since this use case is a bit obscure, I don't think it rises to the level of its own blt smoke test.  If we want to test it should be folded into the gtest test (per haps if-defed, etc)